lib/plugins/Label_8E.ts

```diff
@@ -0,0 +1,58 @@
+import { DecoderPlugin } from '../DecoderPlugin';
+
+// ETA
+export class Label_8E extends DecoderPlugin {
+  name = 'label-8E';
+
+  qualifiers() { // eslint-disable-line class-methods-use-this
+    return {
+      labels: ["8E"],
+    };
+  }
+
+  decode(message: any, options: any = {}) : any {
+    const decodeResult: any = this.defaultResult;
+    decodeResult.decoder.name = this.name;
+    decodeResult.formatted.description = 'ETA Report';
+    decodeResult.message = message;
+
+    // Style: EGSS,1618
+    // Match: arrival_icao,current_time
+    const regex = /^(?<arrival_icao>\w{4}),(?<current_time>\d{4})$/;
+    const results = message.text.match(regex);
+    if (results) {
+      if (options.debug) {
+        console.log(`Label 8E ETA: groups`);
+        console.log(results.groups);
+      }
+
+      
+
+      decodeResult.raw.current_time = new Date();
+      decodeResult.raw.current_time.setUTCHours(results.groups.current_time.substr(0, 2), results.groups.current_time.substr(2, 2));
+
+      if(new Date().getUTCHours() > decodeResult.raw.current_time.getUTCHours()) { // Check if ETA is in the past, which would imply it is for the next day
```

Comment:
All of the samples I have seen (exclusively easyjet and their family of airlines) are all sent at a minimum ~20mins in advanced of the arrival which should negate the need to have a buffer given that time from transmission to being decoded will usually be in the seconds.

It would be useful to have the date of message passed through instead of relying on the current date, but that's more of a bigger picture thing IMO.

Reply:
Thanks for the clarification. Sounds good.